.vscode-template/README.md

```diff
@@ -4,6 +4,22 @@ Recommended workspace settings for [vscode]. **PLEASE** keep this file clean and
 
 Duplicate and rename this folder as ``.vscode``.
 
+``settings.json`` contains the predefined settings for the code editor and other recommended extensions. Please don't change the ones regarding code formatting.
+
+The recommended extensions are:
+- Docker extension
+- Python extension
+- vscode-icons extension (cool icons on the file browser)
+- Babel JavaScript
+
+Install those with the following commands:
+```
+code --install-extension PeterJausovec.vscode-docker
+code --install-extension robertohuertasm.vscode-icons
+code --install-extension ms-python.python
+code --install-extension mgmcdermott.vscode-language-babel
+```
+If VSCode was already running, you need to reload to see them installed. For that you can hit ``Ctrl+Shift+P`` and execute the command ``Reload Window``.
```

Comment:
Are you sure about this? Latest release notes (1.31.1) said exactly the opposite
![image](https://user-images.githubusercontent.com/32402063/53268526-9fa0bc00-36e6-11e9-92c1-dcd045f74627.png)

Reply:
Yes, it was necessary to show the extensions in my 1.31.1 build. At least a window reload..